Dockerfile-full

```diff
@@ -0,0 +1,32 @@
+FROM golang:1.9
+WORKDIR /go/src/github.com/reportportal/service-ui/
+ARG service
+ARG version
+COPY ./Makefile ./
+COPY ./glide.yaml ./
+COPY ./glide.lock ./
```

Comment:
Let's give docker a chance to cache golang's dependencies 
Copy makefile and glide before to be able to cache vendor


COPY Makefile ./
RUN make get-build-deps

COPY glide.yaml glide.lock ./
RUN make vendor